src/tests/data/integrationHappyPathPayload.json

```diff
@@ -1,22 +1,27 @@
 {
+    "yotiMockID": "0000",
     "shared_claims": {
         "name": [
             {
                 "nameParts": [
                     {
-                        "value": "Kenneth",
+                        "value": "API",
```

Comment:
these should be working as we have added this specific user for integration testing with IPVCore

Reply:
Changes removed.